apps/ledger-live-mobile/src/newArch/features/WalletSync/hooks/useSyncWithQrCode.ts

```diff
@@ -75,20 +72,20 @@ export const useSyncWithQrCode = () => {
         return true;
       } catch (e) {
         if (e instanceof InvalidDigitsError) {
-          setCurrentStep(Steps.SyncError);
+          dispatch(setLedgerSyncActivateStep(Steps.SyncError));
```

Comment:
Could be more readable to create a common function maybe called `change step` that would take the wanted Step as param 
It would make it more concise in all the files